CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Unreleased
 
 - [...]
+- **[BREAKING CHANGE]** Complete refactoring of `ItemChoice` to now use `Item` as its base component.
```

Comment:
It's internal, it shouldn't be declared as BREAKING (you already said that the props have been changed)

Reply:
Fixed!